core/templates/pages/classroom-admin-page/classroom-admin-page.component.ts

```diff
@@ -601,7 +761,19 @@ export class ClassroomAdminPageComponent implements OnInit {
     );
   }
 
+  saveClassroomValidationErrors(): string[] {
+    return this.classroomAdminDataService.getSaveClassroomValidationErrors();
+  }
+
+  canPublishTheClassroom(): boolean {
+    return this.validationErrors.length === 0;
```

Comment:
Can we inline this in the template logic (i.e. not use a function)?

Reply:
yes it's just used in once place. Done removed it.